2024-11-28 17:16:04,468 file.manager INFO 2.0.0
2024-11-28 17:16:04,468 file.manager INFO Copyright (C) 2003-2022, Noriaki Ando and OpenRTM development team,
2024-11-28 17:16:04,468 file.manager INFO   Intelligent Systems Research Institute, AIST,
2024-11-28 17:16:04,468 file.manager INFO Copyright (C) 2022, Noriaki Ando and OpenRTM development team,
2024-11-28 17:16:04,468 file.manager INFO   Industrial Cyber-Physical Research Center, AIST,
2024-11-28 17:16:04,468 file.manager INFO   All right reserved.
2024-11-28 17:16:04,468 file.manager INFO Manager starting.
2024-11-28 17:16:04,468 file.manager INFO Starting local logging.
2024-11-28 17:16:04,471 file.manager.namingmanager INFO NameServer connection succeeded: corba/localhost
2024-11-28 17:16:08,559 file.ManagerServant INFO Master manager not found
2024-11-28 17:16:08,560 file.LocalServiceAdmin INFO All the local services are enabled.
2024-11-28 17:16:08,563 file.manager INFO Component type conf file: calculator_temperature.conf loaded.
2024-11-28 17:16:08,564 file.ec_base INFO setRate(1000000.000000) done
2024-11-28 17:16:08,564 file.ec_base INFO setRate(1000.000000) done
2024-11-28 17:16:08,564 file.rtobject INFO 1 execution context was created.
2024-11-28 17:16:08,566 file.rtobject INFO Initial active configuration set is default.
2024-11-28 17:16:22,774 file.ManagerServant INFO Master manager not found
2024-11-28 17:16:32,784 file.ManagerServant INFO Master manager not found
2024-11-28 17:16:42,866 file.ManagerServant INFO Master manager not found
2024-11-28 17:16:52,850 file.ManagerServant INFO Master manager not found
2024-11-28 17:17:02,895 file.ManagerServant INFO Master manager not found
2024-11-28 17:17:12,994 file.ManagerServant INFO Master manager not found
2024-11-28 17:17:13,642 file.result_t WARNING Traceback (most recent call last):
  File "C:\Users\enshu\AppData\Local\Programs\Python\Python310\lib\site-packages\OpenRTM_aist\PortBase.py", line 2305, in checkPorts
    if port._non_existent():
  File "C:\Users\enshu\AppData\Local\Programs\Python\Python310\lib\site-packages\omniORB\CORBA.py", line 559, in _non_existent
    return self._obj.nonExistent()
omniORB.CORBA._omni_sys_exc: CORBA.TRANSIENT(omniORB.TRANSIENT_ConnectFailed, CORBA.COMPLETED_NO)

2024-11-28 17:17:13,642 file.result_t WARNING Dead connection: 128ef60d-ad61-11ef-b42a-f0f7b08dcc27
